docs/.vitepress/config.ts

```diff
@@ -37,6 +27,10 @@ export default defineConfig({
             { text: 'GitCheck', link: '/gitcheck/introduction' },
         ],
 
+        search: {
+            provider: 'local'
+        },
+
```

Comment:
O ile pamiętam to @vLuckyyy miał jakieś zastrzeżenia do natywnego systemu search'owania